apps/opik-frontend/src/components/pages/ProjectsPage/ProjectsPage.tsx

```diff
@@ -89,7 +91,15 @@ const ProjectsPage: React.FunctionComponent = () => {
     {
       workspaceName,
       search,
-      sorting: sortedColumns,
+      sorting: sortedColumns.map((column) => {
+        if (column.id === "last_updated_at") {
+          return {
+            ...column,
+            id: "last_updated_trace_at",
+          };
+        }
```

Comment:
So we have to "duplicate" the `accessorFn` logic here? we can discuss later this one